src/wazuh_db/wdb.c

```diff
@@ -620,7 +620,7 @@ int wdb_create_file(const char *path, const char *source) {
         break;
     }
 
-    if (chmod(path, 0660) < 0) {
```

Comment:
The function `wdb_create_agent_db2` will call `wdb_create_profile` and then this code that you are modifying. The thing is that at the end of `wdb_create_agent_db2` the function will attempt to `chmod` again to `0640`. Just wondering whether we should keep the `chmod` in `wdb_create_agent_db2` or just this one. Could we have an upgrade issue if we remove the `chmod` in `wdb_create_agent_db2`?

Reply:
Not sure about that possible issue, but it is true that the second **chmod** was already there before, and it's harmless.

Reply:
Not sure about that upgrade issue, but the second **chmod** was already there, and it is harmless.